### PR TITLE
Included GAMESS-US, NAMD and Amber easyconfigs

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

File: easybuild/easyconfigs/a/Amber/Amber-14-intel-2017.01-AmberTools-15-patchlevel-13-13.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#
+# modified by Stephane Thiell (Stanford University) for Amber v14
+#
+# author: Benjamin P. Roberts (University of Auckland)
+#
+# based on work by Marios Constantinou (University of Cyprus)
+#
+name = 'Amber'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/a/Amber/Amber-14-intel-2017.01-AmberTools-15-patchlevel-13-13.eb]: This software is installed so keep this easyconfig

File: easybuild/easyconfigs/a/Amber/Amber-14-intelcuda-2017.01-AmberTools-15-patchlevel-13-13.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#
+# modified by Stephane Thiell (Stanford University) for Amber v14
+#
+# author: Benjamin P. Roberts (University of Auckland)
+#
+# based on work by Marios Constantinou (University of Cyprus)
+#
+name = 'Amber'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/a/Amber/Amber-14-intelcuda-2017.01-AmberTools-15-patchlevel-13-13.eb]: remove this easyconfig it is not installed

File: easybuild/easyconfigs/a/Amber/Amber-14-intel-2017.01-AmberTools-15-patchlevel-13-13.eb (hunk 1 above)
Comment (shahzebsiddiqui) [reply easybuild/easyconfigs/a/Amber/Amber-14-intel-2017.01-AmberTools-15-patchlevel-13-13.eb]: http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Amber-14-intel-2017.01-AmberTools-15-patchlevel-13-13-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/a/Automake/Automake-1.15-intelcuda-2017.01.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+##
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/a/Automake/Automake-1.15-intelcuda-2017.01.eb]: This is installed http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Automake-1.15-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/a/Autotools/Autotools-20150215-intelcuda-2017.01.eb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/a/Autotools/Autotools-20150215-intelcuda-2017.01.eb]: This is installed http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/Rpm/easybuild/x86_64/rhel/7.3/Autotools-20150215-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/b/bzip2/bzip2-1.0.6-intelcuda-2017.01.eb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+name = 'bzip2'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/b/bzip2/bzip2-1.0.6-intelcuda-2017.01.eb]: This is installed http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/bzip2-1.0.6-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/c/cURL/cURL-7.49.1-intelcuda-2017.01.eb [hunk 6]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/c/cURL/cURL-7.49.1-intelcuda-2017.01.eb]: Keep this easyconfig, rpm is at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/cURL-7.49.1-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-intel-2017.01.eb [hunk 7]
```diff
@@ -0,0 +1,25 @@
+name = 'FFTW'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-intel-2017.01.eb]: Keep this easyconfig, rpm is at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/FFTW-3.3.6-intel-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20170420-R1-intel-2017.01.eb [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+# Copyright:: Copyright 2012-2017 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+#
+# Authors::
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Ward Poelmans <wpoely86@gmail.com>
+# * Oliver Stueker <oliver.stueker@ace-net.ca>
+# License::   MIT/GPL
+##
+name = 'GAMESS-US'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20170420-R1-intel-2017.01.eb]: The rpm is at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/GAMESS-US-20170420-R1-intel-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/g/GMP/GMP-6.1.1-intelcuda-2017.01.eb [hunk 9]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/g/GMP/GMP-6.1.1-intelcuda-2017.01.eb]: This is okay, I see rpm at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/GMP-6.1.1-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intelcuda-2017.01.eb [hunk 10]
```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-intelcuda-2017.01.eb]: This is okay, rpm is at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/HDF5-1.8.16-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/l/libffi/libffi-3.2.1-intelcuda-2017.01.eb [hunk 11]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'libffi'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/l/libffi/libffi-3.2.1-intelcuda-2017.01.eb]: this is okay, rpm is at http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/libffi-3.2.1-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/l/libreadline/libreadline-6.3-intelcuda-2017.01.eb [hunk 12]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libreadline'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/l/libreadline/libreadline-6.3-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/libreadline-6.3-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/l/libtool/libtool-2.4.6-intelcuda-2017.01.eb [hunk 13]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'libtool'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/l/libtool/libtool-2.4.6-intelcuda-2017.01.eb]: this is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/libtool-2.4.6-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017.01-mpi.eb [hunk 14]
```diff
@@ -0,0 +1,24 @@
+name = 'NAMD'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017.01-mpi.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/NAMD-2.12-intel-2017.01-mpi-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-intelcuda-2017.01.eb [hunk 15]
```diff
@@ -0,0 +1,17 @@
+name = 'netCDF-Fortran'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/netCDF-Fortran-4.4.4-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1-intelcuda-2017.01.eb [hunk 16]
```diff
@@ -0,0 +1,33 @@
+name = 'netCDF'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/netCDF-4.4.1-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/p/Python/Python-2.7.12-intelcuda-2017.01.eb [hunk 17]
```diff
@@ -0,0 +1,126 @@
+name = 'Python'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/p/Python/Python-2.7.12-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Python-2.7.12-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/s/SQLite/SQLite-3.13.0-intelcuda-2017.01.eb [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.13.0-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/SQLite-3.13.0-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/s/Szip/Szip-2.1-intelcuda-2017.01.eb [hunk 19]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'Szip'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/s/Szip/Szip-2.1-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Szip-2.1-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/t/Tcl/Tcl-8.6.5-intelcuda-2017.01.eb [hunk 20]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tcl'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/t/Tcl/Tcl-8.6.5-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Tcl-8.6.5-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/t/Tk/Tk-8.6.5-intelcuda-2017.01.eb [hunk 21]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/t/Tk/Tk-8.6.5-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/Tk-8.6.5-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

File: easybuild/easyconfigs/z/zlib/zlib-1.2.8-intelcuda-2017.01.eb [hunk 22]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
```
Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.8-intelcuda-2017.01.eb]: This is okay http://hpcv04.pfizer.com:8081/artifactory/webapp/#/artifacts/browse/tree/General/easybuild/x86_64/rhel/7.3/zlib-1.2.8-intelcuda-2017.01-eb_3.3.1-2.x86_64.rpm

**Comment by algiar** [comment]
Hi @shahzebsiddiqui , I have removed the Amber-14-intelcuda-2017.01-AmberTools-15-patchlevel-13-13.eb from the PR. I think you could resolve it, rename the branch to easybuild-2017 and stop using it.

**Comment by shahzebsiddiqui** [comment]
@jordiblasco i recall you had to implement custom easyblocks for some apps if you have this somewhere can you send PR to upstream and also https://github.com/PfizerRD/easybuild-easyblocks so we can reuse them for next release.

@omula

**Pull request merged**